andela/travel_tool_front
# #160124779 Make notification pane toggle

#### What does this PR do?
This PR ensures that if the notification icon gets clicked multiple times, the notification icon gets toggled

#### Description of Task to be completed?
- write tests for the situation where the notification icon is clicked twice
- move `onNotificationToggle` method from RequestsPage.jsx to Base/index.jsx because it is common to requests and approvals
- modify the `onNotificationToggle` method such that the states `hideNotificationPane` and `hideSideBar` toggle when it is called

#### How should this be manually tested?
- Clone the repo - `https://github.com/andela/travel_tool_front.git`
- Switch to the repo directory - `cd travel_tool_front`
- Check out this branch - `git checkout bg-notification-toggler-fix-160124779`
- Install dependencies and start the app - `make start`
- Navigate to `localhost:3000/requests` and click the notification icon multiple times
- It should toggle the notification pane

#### Any background context you want to provide?
If you have any issues while trying to set up the app, pls check the `Setup` section of the  [README](https://github.com/andela/travel_tool_front#setup)

#### What are the relevant pivotal tracker stories?
[#160124779]( https://www.pivotaltracker.com/story/show/160124779)

#### Screenshots (if appropriate)
None

#### Questions:
